microraiden/microraiden/examples/echo_client.py

```diff
@@ -10,45 +10,43 @@
 import logging
 import requests
 
-from microraiden.config import CHANNEL_MANAGER_ADDRESS
-
 
 @click.command()
 @click.option(
-    '--key-path',
+    '--private-key',
     required=True,
-    help='Path to private key file.',
+    help='Path to private key file or a hex-encoded private key.',
```

Comment:
It's not the smartest idea to expose privkey to anyone who can run `ps -aux` on your system :)

Reply:
Still, developers should be able to for quick testing which is probably the case 99% of the time this example runs. I doubt people use their real mainnet wallet for these kind of tests. Even then, they'd have to get access to their hex private key.